app/controllers/decidim_controller.rb

```diff
@@ -1,3 +1,10 @@
 # frozen_string_literal: true
 class DecidimController < ActionController::Base
+  before_action :set_raven_context
+
+  def set_raven_context
+    return unless Rails.env.production?
```

Comment:
I'd use something at application secrets, like the sentry key or if sentry is enabled.